app/controllers/ChannelTestsController.scala

```diff
@@ -151,4 +156,130 @@ abstract class ChannelTestsController[T <: ChannelTest[T] : Decoder : Encoder](
     }
   }
 
+  /**
+    * Handlers for test list ordering
+    */
```

Comment:
these handlers for test list ordering use the existing S3 locking mechanism, so they're almost the same as the old `lock`/`unlock` methods above